### DISCUSSION TITLE
Show/Hide all code with one click

### DISCUSSION THREAD

**Discussion by rpherman** [fonsp/Pluto.jl#1037] [Questions about using Pluto] [answered]
Is there a way to select all the 'eye' toggles to show/hide all code with one click?
I usually have to go through each cell and unhide code after printing a report without code. Thanks!

**Comment by fonsp** [comment, marked as answer]
If you have no element focused, then `Ctrl+A` will select all cells, you can then press any of the eyes to toggle all at once.

You can also drag-select from the right of the cells.

![select edit multiple](https://user-images.githubusercontent.com/6933510/113166976-8ebac100-9243-11eb-93f9-3893875a3ce6.gif)

**Comment by rpherman** [comment]
Thanks! Always simple once you see the answer.
